shared/mad/codeql/mad/ModelValidation.qll

```diff
@@ -33,7 +33,7 @@ module KindValidation<KindValidationConfigSig Config> {
           "bean-validation", "fragment-injection", "groovy-injection", "hostname-verification",
           "information-leak", "intent-redirection", "jexl-injection", "jndi-injection",
           "mvel-injection", "ognl-injection", "pending-intents", "response-splitting",
-          "template-injection", "xpath-injection", "xslt-injection",
+          "trust-boundary-violation", "template-injection", "xpath-injection", "xslt-injection",
```

Comment:
Should this sink kind be added to the list of supported kinds in [customizing-library-models-for-java.rst](https://github.com/github/codeql/blob/1c64fb16f1d0212e0e10ad3547e5cefad273a7a5/docs/codeql/codeql-language-guides/customizing-library-models-for-java.rst#sinkmodelpackage-type-subtypes-name-signature-ext-input-kind-provenance)?

Reply:
That makes sense. Are there any other places I should update when adding a new sink kind?

Reply:
That's the only other place that needs updating as far as I'm aware.